tests_thien/testcases/dashboard/login_test.py

```diff
@@ -0,0 +1,38 @@
+from tests_thien.testcases.test_base import TestBase
+from tests_thien.pages.dashboard.login_page import LoginPage
+from tests_thien.pages.dashboard.dashboard_page import DashboardPage
+from tests_thien.utilities import constant, browser_helper
+import pytest
+
+class LoginTest(TestBase):
+    da_login = LoginPage()
+    da_page = DashboardPage()
+
+    @pytest.mark.usefixtures("clean_up_valid_login")
+    def test_001_login_successful(self):
+        # Verify that user can login specific repository successfully via Dashboard login page with correct credentials
+        self.da_login.login(constant.DA_USER,constant.DA_PWD)
```

Comment:
Should use the default value for username and password

Reply:
updated